shells/pipes-shell/web/cache-mgr.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+const ARCS_CACHE_MANAGER = './cache-mgr-sw.js';
+
+// Arcs Cache Manager
+(async function _arcsCacheManager() {
+  if ('serviceWorker' in navigator) {
```

Comment:
It's completely equivalent to what you have here, but IMO this pattern is a little nicer:

```
const serviceWorker = navigator.serviceWorker;
if (!serviceWorker) {
  return;
}
// Use serviceWorker.
```

It doesn't make any difference for plain old JavaScript, but for TypeScript the compiler will help you out if you make a typo with the name, whereas with a raw string it won't.

Reply:
Good to know this difference, will change.

Reply:
Done!